hinted_handoff_test.py

```diff
@@ -0,0 +1,128 @@
+import os
+import re
+import time
+
+from aexpect.utils.process import CmdError
+from avocado import main
+
+from sdcm.db_stats import PrometheusDBStats
+from sdcm.tester import ClusterTester
+from sdcm.utils import retrying
+
+
+class HintedHandoffTest(ClusterTester):
+    """
+       :avocado: enable
+    """
+    def __init__(self, *args, **kwargs):
+        super(HintedHandoffTest, self).__init__(*args, **kwargs)
+        self.scylla_dir = "/var/lib/scylla"
```

Comment:
May be we need to define this default Scylla path in the cluster class, for example, and use it, instead of write hard coded in the different tests?

Reply:
Agreed.